docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/LightGbm.cs

```diff
@@ -35,8 +35,8 @@ public static void Example()
             var pipeline = mlContext.Transforms.Concatenate("Features", featureNames)
                            .Append(mlContext.Regression.Trainers.LightGbm(
                                             labelColumnName: labelName,
-                                            numLeaves: 4,
-                                            minDataPerLeaf: 6,
+                                            leafCount: 4,
```

Comment:
```suggestion
                                            numberOfLeaves: 4,
```
as suggested in #2613. I can follow you but we need to change other places where #2613 is already applied.